pkg/event/event.go

```diff
@@ -83,7 +82,6 @@ func (e Event) String() string {
 	}
 
 	b.WriteString("  data: \r\n")
-	b.WriteString("  version: " + e.Data.Version + "\r\n")
```

Comment:
Do not remove data.version from event. It is event data schema version, not REST API verison. See any example of event in Table 34 in O-RAN spec:  "data": {"version" : "1.0",...} is needed in an event. 
https://specifications.o-ran.org/download?id=670

Reply:
Yes, I got it. I remove the PR.